cloudera/datadog_checks/cloudera/api/api_v7.py

```diff
@@ -68,7 +69,7 @@ def _collect_clusters(self):
                     cluster_name = key
                     tags = self._collect_cluster_tags(item)
                     submit(self._collect_cluster_metrics, cluster_name, tags)
-                    submit(self._collect_hosts, cluster_name)
+                    submit(self._collect_hosts, cluster_name, config)
```

Comment:
Just a small nit, but I'm wondering if we can rename `config` to `cluster_discovery_config` or similar to make it distinct from `self._check.config`? Similar to how you rename `key` to `cluster_name`.